src/Surfnet/StepupMiddleware/ApiBundle/Controller/CommandController.php

```diff
@@ -28,13 +30,16 @@
 
 class CommandController extends Controller
 {
-    public function handleAction(Command $command, Request $request)
+    public function handleAction(Command $command, Metadata $metadata, Request $request)
     {
         /** @var \Monolog\Logger $logger */
         $logger = $this->get('logger');
-
         $logger->notice(sprintf('Received request to process Command "%s"', $command));
 
+        /** @var MetadataEnricher $metadataEnricher */
+        $metadataEnricher = $this->get('surfnet_stepup_middleware_command_handling.metadata_enricher.actor');
+        $metadataEnricher->enrichWithMetadata($metadata);
```

Comment:
`setMetadata`? `populateMetadata`? Don't know, but you're not enriching anything here :wink: